rules/integrations/aws_bedrock/aws_bedrock_high_confidence_misconduct_blocks_detected.toml

```diff
@@ -1,22 +1,22 @@
 [metadata]
 creation_date = "2024/05/05"
 maturity = "production"
-updated_date = "2024/11/05"
+updated_date = "2024/11/21"
 min_stack_comments = "ES|QL rule type is still in technical preview as of 8.13, however this rule was tested successfully; integration in tech preview"
 min_stack_version = "8.13.0"
 
 [rule]
 author = ["Elastic"]
 description = """
-Detects repeated high-confidence 'BLOCKED' actions coupled with specific violation codes such as 'MISCONDUCT',
-indicating persistent misuse or attempts to probe the model's ethical boundaries.
+Detects repeated high-confidence 'BLOCKED' actions coupled with specific 'Content Filter' policy violation having codes such as 'MISCONDUCT', 
+'HATE', 'SEXUAL', INSULTS', 'PROMPT_ATTACK', 'VIOLENCE' indicating persistent misuse or attempts to probe the model's ethical boundaries.
 """
 false_positives = ["New model deployments.", "Testing updates to compliance policies."]
 from = "now-60m"
 interval = "10m"
 language = "esql"
 license = "Elastic License v2"
-name = "Unusual High Confidence Misconduct Blocks Detected"
+name = "Unusual High Confidence Content Filter Blocks Detected"
```

Comment:
How is telemetry looking on these? I would personally bump severity down to medium for these rules for now.

Reply:
The integration is still in Beta so i am not sure on the adoption and telemetry. 

This will only be alerted on HIGH Confidence blocks, we are already filtering MEDIUM and Low ones via gen_ai.policy.confidence LIKE "HIGH"

Reply:
I was talking about rule severity here, not the confidence blocks. The rules all have severity high, which I think might be a bit too much. I would push them to medium.

Reply:
Ah ok, I was more thinking on Confidence Blocks, 

yeah given we have increased scope from MISCONDUCT, we can bump down to medium!. Look back into telemetry and then regroup major contents into HIGH and rest to medium or allow customisations from user end!.